.github/workflows/php.yml

```diff
@@ -0,0 +1,70 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ develop, master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        php: ["7.2", "7.3", "7.4", "8.0", "8.0"]
```

Comment:
Should the `composer_update_flags` params be part of the Matrix? On travis we are running a total of 10 jobs for the php version 7.1, 7.2, 7.3, 7.4, 8.0 and with no composer update flags and with the `--prefer-lowest --prefer-stable` update flags. only asking as Im new to Github Actions

Reply:
The reason I did it like this was so that it would be possible to add one run without the composer update flags.
As I believe was intended with travis on this line. https://github.com/JsonMapper/JsonMapper/blob/84596e1a54e5698df9801a7d9adaa30996ac5df3/.travis.yml#L21